app/controllers/inventory_foods_controller.rb

```diff
@@ -0,0 +1,75 @@
+class InventoryFoodsController < ApplicationController
+  load_and_authorize_resource
+  before_action :set_inventory_food, only: %i[show edit update destroy]
+
+  # GET /inventory_foods or /inventory_foods.json
+  def index
+    @inventory_foods = InventoryFood.all
+  end
+
+  # GET /inventory_foods/1 or /inventory_foods/1.json
+  def show; end
+
+  # GET /inventory_foods/new
+  def new
+    @inventory_food = InventoryFood.new
```

Comment:
* Kindly fix this error i get when i try to create new inventory food 
![image](https://user-images.githubusercontent.com/52497246/219844984-681fd7dc-e3af-45ec-8062-d60cd6361b8c.png)

Reply:
@Davidosky007 thanks for the review
I have fix the problem